Classes/Command/JobCommandController.php

```diff
@@ -84,6 +90,12 @@ public function workCommand($queue, $exitAfter = null, $limit = null, $verbose =
             }
             try {
                 $message = $this->jobManager->waitAndExecute($queue, $timeout);
+                $this->jobManager->interruptMe();
```

Comment:
Why is this method called here? This will be invoked for every successfully processed message in the queue. Does that make sense?

Reply:
Yeah. By calling this message, it allows the jobManager to be interrputed in this (and only in this) moment. And this is definitely a safe moment, because the last message has been "completed" (maybe by timeout, or successful execution). Usually it will probably simply do nothing.

Reply:
OK, I think my confusion came from the name since this doesn't always actually interrupt obviously.